src/main/host/descriptor/socket/unix.rs

```diff
@@ -330,16 +336,19 @@ struct ConnOrientedConnected {
     bound_addr: Option<nix::sys::socket::UnixAddr>,
     peer_addr: nix::sys::socket::UnixAddr,
     send_buffer: Arc<AtomicRefCell<SharedBuf>>,
+    writer_handle: WriterHandle,
     #[allow(dead_code)]
-    send_buffer_event_handle: BufferHandle,
+    send_buffer_handle: BufferHandle,
 }
 struct ConnOrientedClosed {}
 
 struct ConnLessInitial {
     bound_addr: Option<nix::sys::socket::UnixAddr>,
     peer_addr: Option<nix::sys::socket::UnixAddr>,
     send_buffer: Option<Arc<AtomicRefCell<SharedBuf>>>,
-    send_buffer_event_handle: Option<BufferHandle>,
+    writer_handle: Option<WriterHandle>,
+    #[allow(dead_code)]
```

Comment:
Needed? Or left over from an intermediate state? If it is needed, a brief comment about why and/or when to remove it would be helpful

Reply:
It seems like it's recommended to use an underscore in this case rather than a `dead_code` warning, so I did that instead. https://github.com/rust-lang/rust/issues/21775